content/cpp/concepts/vectors/vectors.md

```diff
@@ -148,6 +148,21 @@ std::cout << "The last element is: " << v[n - 1];
 // Output: The last element is: 5
 ```
 
+### `.emplace_back()`
+
+This method also pushes elements to the back of a vector, but instead of creating a temporary object it directly creates object in the vector itself:
```

Comment:
```suggestion
This method also pushes elements to the back of a vector, but instead of creating a temporary object it directly creates an object in the vector itself.
```